policy_set_integration_test.go

```diff
@@ -284,20 +284,20 @@ func TestPolicySetsRead(t *testing.T) {
 		require.NoError(t, err)
 
 		opts := &PolicySetReadOptions{
-			Include: "current-version,newest-version",
+			Include: []PolicySetIncludeOps{PolicySetCurrentVersion, PolicySetNewestVersion},
 		}
-		ps, err = client.PolicySets.ReadWithOptions(ctx, psTest.ID, opts)
```

Comment:
I changed this because there was a race condition here, making this integration test fail occasionally.